extensions/guacamole-auth-radius/src/licenses/bundled/README

```diff
@@ -0,0 +1,4 @@
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
```

Comment:
Okay, I got a response on the LEGAL JIRA issue I opened up.  See it, here:
https://issues.apache.org/jira/browse/LEGAL-299

Based on the response, there, I think we're okay - I have one follow-up that I'll ask on there, but it seems like we're falling very narrowly in the scope of what's okay to do in ASF projects with (L)GPL licenses.